db/schema.rb

```
@@ -12,686 +10,753 @@
+    t.string "diagnosis_reference"
+    t.string "diagnosis_level"
+    t.string "record_source"
+    t.date "first_date_used"
+    t.date "last_date_used"
+    t.integer "record_count"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["ite_client_id"], name: "index_ite_client_diagnoses_on_ite_client_id"
+  end
+
+  create_table "ite_client_mdm_mappings", force: :cascade do |t|
```

RuboCop (1.75.5)

[convention] Prefer single-quoted strings when you don't need string interpolation or special symbols.

(Style/StringLiterals)